com.unity.UnityHub.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: com.unity.UnityHub
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: start-unityhub
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - '--allow=multiarch'
+  - '--device=dri'
+  - '--filesystem=host'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--talk-name=org.freedesktop.Flatpak'
+  - '--talk-name=org.gnome.Software'
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '18.08'
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: '18.08'
+modules:
+  - name: compat
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p /app/lib/i386-linux-gnu /app/lib/debug/lib/i386-linux-gnu'
+
+  - shared-modules/glu/glu-9.0.0.json
+
+  - name: unappimage
```

Comment:
AppImages can be self-extracted with `--appimage-extract`, why use additional tool?

Reply:
OK, you actually explained it somewhere above in the discussion. Would be nice to have a comment in the manifest later.